OneCommunityGlobal/HighestGoodNetworkApp
# Vishala adding surge dep

# Description
Added surge dependency

## Main changes explained:
- Added dev dependencies (surge)

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Check App functionality
